Modules/Module_Import/Import/xASL_imp_DCM2NII_Subject_SortASLVolumes.m

```diff
@@ -146,6 +146,21 @@
         elseif  ~isempty(strfind(scan_name,'M0'))
             [nii_files,ASLContext] = xASL_bids_MergeNifti(nii_files, 'M0', niiTable);
         end
+	else
+		% In case of a single file, we need to check issues with orientation
+		% Check for a special case of Siemens MOSAIC files without Phoenix field
+		if isfield(tmpJSON, 'Manufacturer') && strcmpi(tmpJSON.Manufacturer, 'Siemens') && isfield(tmpJSON, 'ImageType') && ~isempty(regexpi(tmpJSON.ImageType, 'MOSAIC')) && (~isfield(tmpJSON, 'PhoenixProtocol') || isempty(tmpJSON.PhoenixProtocol))
+			niftiData = xASL_io_ReadNifti(nii_files{1});
```

Comment:
Should we have a warning if there are more than 1 nii_files?

Reply:
Not needed - we're in an IF branch that only has 1 nii_files. So we only need {1} to move from cell to character array, but it cannot happen here that we have more than 1 nii_files at this line.